packages/expo-modules-core/src/ts-declarations/global.ts

```diff
@@ -62,4 +62,27 @@ declare global {
    * This object is not available in projects without the `expo` package installed.
    */
   var expo: ExpoGlobal;
+
+  const process: {
+    env: {
+      NODE_ENV: string;
+      /** Used in `@expo/metro-runtime`. */
+      EXPO_DEV_SERVER_ORIGIN?: string;
+      /** Enables static rendering entry point on web. */
+      EXPO_PUBLIC_USE_STATIC?: string;
+
+      EXPO_ROUTER_IMPORT_MODE?: string;
+      EXPO_ROUTER_ABS_APP_ROOT?: string;
+      EXPO_ROUTER_APP_ROOT?: string;
```

Comment:
cc @EvanBacon
We might want to move router-specific env to expo-router or the `expo` package. They are not part of Expo Modules API in general.

Reply:
They're injected by Metro > `babel-preset-expo` so all projects get them. Probably we should rename these in the future to be `_EXPO` or something.